### ISSUE TITLE
About the result of graph classification

### ISSUE THREAD

**Issue by dongZheX** [graph-star-team/graph_star#14]
Hello, thanks for the code.

I notice that you split train/val/test dataset in graph classification task. But, in the training process, it seems that you don't use the result of val to choose best epoch for getting final result of test set. I think it would be better if you choose the the result of test set in the epoch having mininum validation loss as the final result in one experiment.(Baseline diffpool does like this).

In fact, in the area of graph classification, many works are compared in different experimental setting which leads to unfair result. Now, I am a little confused and don't know how to choose a correct experimental settings.

Looking forward your reply.
